ipa-core/src/net/client/mod.rs

```diff
@@ -186,17 +191,18 @@ impl MpcHelperClient {
             (
                 HttpsConnectorBuilder::new()
                     .with_native_roots()
+                    .unwrap()
                     .https_or_http()
                     .enable_http2()
                     .wrap_connector(make_http_connector()),
                 auth_header,
             )
         } else {
-            let builder = rustls::ClientConfig::builder().with_safe_defaults();
+            let builder = rustls::ClientConfig::builder();
             let client_config = if let Some(certificate) = peer_config.certificate {
                 let cert_store = {
                     let mut store = RootCertStore::empty();
-                    store.add(&certificate).unwrap();
+                    store.add(certificate).expect("Error adding certificate");
```

Comment:
+1 on changing `.unwrap() to `expect`. One suggestion is to use the form "should" as advised in [std](https://doc.rust-lang.org/std/result/enum.Result.html#method.expect)